include/glow/Base/Image.h

```diff
@@ -91,6 +93,21 @@ Tensor readPngImageAndPreprocess(llvm::StringRef filename,
                                  ImageChannelOrder imageChannelOrder,
                                  ImageLayout imageLayout,
                                  bool useImagenetNormalization);
+
+/// Loads and normalizes all PNGs into a tensor in the NHWC format with the
+/// requested channel ordering.
+/// \param filenames list of filenames to read.
+/// \param inputImageData Tensor to save the resulting output.
+/// \param imageNormMode normalize values to this range.
+/// \param imageChannelOrder the order of color channels.
+/// \param imageLayout the order of dimensions (channel, height, and width).
+/// \param useImagenetNormalization use special normalization for Imagenet.
+void loadImagesAndPreprocess(const llvm::cl::list<std::string> &filenames,
```

Comment:
You should be able to use `llvm::ArrayRef<std::string> filenames` instead here, and then you won't have to include `CommandLine.h` in `Image.h`.